### PR TITLE
Correct DOI

### PR DESCRIPTION
**Pull request by davidwilby**
For some reason the last release created a new Zenodo record rather than adding the new version to the existing record. This replacement DOI points to the most recent version on the new record, which is linked to the repo via my Zenodo account.

Could alternatively merge to master, rather than dev.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Teamware is developed by the [GATE](https://gate.ac.uk) team, an academic resear
 ## Citation
 For published work that has used Teamware, please cite this repository. One way is to include a citation such as:
 
-> Karmakharm, T., Wilby, D., Roberts, I., & Bontcheva, K. (2022). GATE Teamware (Version 0.1.4) [Computer software]. https://github.com/GateNLP/gate-teamware
+> Karmakharm, T., Wilby, D., Roberts, I., & Bontcheva, K. (2022). GATE Teamware (Version 2.1.0) [Computer software]. https://github.com/GateNLP/gate-teamware
```
Comment (twinkarma) [inline README.md:66]: Should we add something in the version.py to update the version number here or at least a check as a reminder?

**Comment by davidwilby** [comment]
> So what was causing the new DOI to be created in the first place?

I had previously created the Zenodo record manually, then linked the repo with Zenodo, which creates a new record and DOI automatically on release, so in this case it made a whole new 'upload' (in zenodo terminology) for release 2.1.0, but now all subsequent records should be grouped together under this doi.

File: README.md (hunk 1 above)
Comment (davidwilby) [reply README.md:66]: Added this to `version.py` in [1ab0cf2](https://github.com/GateNLP/gate-teamware/pull/364/commits/1ab0cf2d9ff53bddb0b548c7ca78f1d478874c08)

**Review by ianroberts — CHANGES_REQUESTED** [review]

File: version.py [hunk 2]
```diff
@@ -40,6 +45,20 @@ def get_package_json_version(file_path: str) -> str:
     js_version = package_json['version']
     return js_version
 
+def get_readme_version(file_path: str) -> str:
+    with open(file_path, 'r') as f:
+        readme_text = f.read()
+    
+    match = re.search(r'\(Version (.*)\)', readme_text)
```
[the remark this answers is not in the document]
Comment (davidwilby) [reply version.py:52]: Thanks @ianroberts - much appreciated. I'm just trying my best at regex here! Cheers

**Pull request merged**